docs/surveys/salary-survey/2022.rst

```diff
@@ -14,7 +14,2069 @@ Write the Docs Salary Survey 2021 Results
+   | Support               |  38 |  7.2% |
+   +-----------------------+-----+-------+
+   | Marketing             |  26 |    5% |
+   +-----------------------+-----+-------+
+   | Other                 |  26 |    5% |
+   +-----------------------+-----+-------+
+
+Of those who selected "Other", responses included project manager, product owner/manager, information architect, and UX writer. 
+
+Team breakdown
+~~~~~~~~~~~~~~
+
+When taking into account respondents who work on a team, work on multiple teams, and those who manage teams, 83.4% of respondents have roles that are team-based. Solo workers make up 15.4%, with the 1.1% selecting "other" mostly reporting some combination of solo and team work. 
+
+.. table:: Team breakdown (2022)
+   :width: 70%
+   :name: tbl-2022-team-breakdown
+
+   +------------------------+-----+-------+
+   | Team Type              | No. | %     |
+   +========================+=====+=======+
+   | Part of a team         | 238 | 45.3% |
+   +------------------------+-----+-------+
+   | Part of multiple teams | 115 | 21.9% |
+   +------------------------+-----+-------+
+   | Manager or team leader | 85  | 16.2% |
+   +------------------------+-----+-------+
+   | Solo workers           | 81  | 15.4% |
+   +------------------------+-----+-------+
+   | Other                  | 6   | 1.1%  |
+   +------------------------+-----+-------+
+
+.. raw:: html
+
+   <figure>
+      <object role="img" aria-label="Team breakdown (2022)" aria-describedby="figure-team-breakdown_desc" type="image/svg+xml" data="/_images/2022-team-breakdown.svg">
+         <p id="figure-team-breakdown_desc">Team breakdown (2022)</p>
+      </object> 
+      <figcaption>Figure: Team breakdown (2022)</figcaption>
+   </figure>
+
+.. figure:: images/2022/2022-team-breakdown.svg
+   :class: hide
+
+Length of time at current organization
+--------------------------------------
+
+.. container:: question
+   
+   .. rubric:: What we asked
+
+   How long have you worked at your current organization?
+   
+   Note:
+   Please select the length of time for your position at your current organization only – your total years of experience in documentation will be covered in the individual demographics section. If you have changed roles at the same organization, please select the length of time that you have been in your current role.
+
+   - Less than 1 year
+   - More than 1 year but less than 2 years
+   - More than 2 years but less than 5 years
+   - More than 5 years but less than 10 years
+   - More than 10 years
+
+Reflecting an increasingly volatile job market, 36.8% of respondents reported being employed at their current organization for less than one year. This marks a significant increase from 31.7% in 2021, 26% in 2020, and just 9% in 2019. Notably, this year marks the first time since the inception of the survey that the number of individuals in new positions exceeds those who have held their current job for a medium-term or extended duration.    
```

Comment:
Inception feels mildly odd here. Since we've been running the survey?
Also double "this marks" and it isn't really the year that marks the first time. 

```suggestion
Reflecting an increasingly volatile job market, 36.8% of respondents reported being employed at their current organization for less than one year. This is a significant increase from 31.7% in 2021, 26% in 2020, and just 9% in 2019. Notably, it is the first time since the survey began that the number of individuals in new positions exceeds those who have held their current job for a medium-term or extended duration.    
```